### PR TITLE
Add basic nix setup

### PR DESCRIPTION
**Pull request by naechtner**

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -1,3 +1,26 @@
...
+name = "turn-events"
+version = "0.1.0"
+description = ""
+authors = ["Sten Aechtner <stean.aechtner@gmail.com>", "Nico Aechtner <nico.aechtner@gmail.com>"]
+license = "license.md"
+readme = "README.md"
+packages = [{include = "turn_events"}]
+
+[tool.poetry.dependencies]
+django = "1.11.29"
```
Comment (naechtner) [inline pyproject.toml:11]: TODO: Update dependencies from `master`

File: .pre-commit-config.yaml [hunk 2]
```diff
@@ -9,24 +9,31 @@ repos:
...
 -   repo: https://github.com/adamchainz/django-upgrade
     rev: 1.13.0
     hooks:
     -   id: django-upgrade
         args: [--target-version, "4.2"]
--   repo: https://github.com/pycqa/isort
-    rev: 5.12.0
-    hooks:
-    -   id: isort
+# Replaced by ruff:
```
Comment (saechtner) [inline .pre-commit-config.yaml:22]: isort gone? 😢

File: README.md [hunk 3]
```diff
@@ -9,6 +9,13 @@ Currently runs on Python 3.9+.
 - Install requirements: `pip install -r requirements.txt`
 - Set up db schemas: `python manage.py migrate`
 
+## Alternative setup using nix
+- Install nix: `curl -L https://nixos.org/nix/install | sh`
+- Install direnv `brew install direnv`
+- Create a file `.envrc` in the project root with the following content: `use flake`
+- Run `direnv allow` and wait for the environment to build
+- Set up db schemas: `python manage.py migrate`
```
Comment (saechtner) [inline README.md:17]:
```suggestion
- Set up db schemas: `./manage.py migrate`
```

**Review by saechtner — APPROVED** [review]

File: .pre-commit-config.yaml (hunk 2 above)
Comment (naechtner) [reply .pre-commit-config.yaml]: Added back in e0151cfbb6da5044a4b3403e75624a4c22786877

**Pull request merged**